template-parts/modules/focus-blocks/blocks.php

```diff
@@ -13,16 +13,19 @@
 
 $block_count = 0;
 
-?><div class="mw-980 flex flex-medium flex-space-between"><?php
+?><div class="mw-980 flex flex-medium flex-space-between">
+<?php
 foreach ( $template_args['blocks'] as $block ) {
 	if ( empty( $block ) || ! is_array( $block ) ) {
 		continue;
 	}
 
-	$block_count++;
+	++$block_count;
```

Comment:
Interesting that it's detecting a standalone statement, which makes no diffference, for pre or post increment